### ISSUE TITLE
Headings.

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#489]
# Templates

Visibility: 2
Evaluation: Partially Supports
Success Criteria: 2.4.6: Headings and Labels
Needs design: 

### Description
The headings should describe the purpose/topic of the section/page. 

On the Templates pages, as you move through the template folders, the pages use the breadcrumbs as the heading. The breadcrumbs should be in a `<nav>` element and each page given an appropriate heading to describe the topic. For example, the breadcrumb might be templates>outreach and the heading might be “Outreach”, “Outreach Templates” or “Outreach folder”. The related Manage page might have a heading like “Manage Outreach folder”

![]()

**Comment by yaelberger-commits** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/489?planning-poker) with ZenHub @amazingphilippe @andrewleith

**Comment by yaelberger-commits** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/489?planning-poker) with ZenHub @adriannelee

**Comment by yaelberger-commits** [comment]
Might be fixed by another issue, but if not tackle next

**Comment by amazingphilippe** [comment]
This was fixed by cds-snc/notification-admin#1244
